packages/analytics/src/integrations/Omnitracking/definitions.ts

```diff
@@ -592,14 +592,34 @@ export const trackEventsMapper: Readonly<OmnitrackingTrackEventsMapper> = {
     actionArea: data.properties?.from,
     productId: getProductId(data.properties),
   }),
-  [EventType.Login]: data => ({
-    tid: 2924,
-    loginType: data.properties?.method,
-  }),
-  [EventType.SignupFormCompleted]: data => ({
-    tid: 2927,
-    loginType: data.properties?.method,
-  }),
+  [EventType.Login]: data => {
+    if (!data.properties?.method) {
```

Comment:
Since this validation code is in login and signup events, what do you think of adding a helper validation function that could be called by both the login and sign up events?

Reply:
fixed